lib/index.js

```diff
@@ -40,9 +43,12 @@ const getPkgName = options => {
 
 const fileProxy = async (options, file) => {
 	const { cwd, cjsDir, esmDir, typesDir } = options
-	const pkgName = pkgCache.has(options)
-		? pkgCache.get(options)
-		: getPkgName(options)
+	const pkgName =
```

Comment:
could u use if/else for this? im not a fan of nested ternaries 😉 btw. was prettier ran over this file during precommit?

Reply:
Yep. This is how the latest version formats ternaries.